src/main/java/callofcactus/GameInitializer.java

```diff
@@ -0,0 +1,76 @@
+package callofcactus;
+
+
+import callofcactus.menu.MainMenu;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+
+import java.rmi.RemoteException;
+
+/**
+ * @author Teun
+ */
+public class GameInitializer extends com.badlogic.gdx.Game {
+
+    private callofcactus.Game game;
+
+    private OrthographicCamera camera;
+    private SpriteBatch batch;
+
+    public GameInitializer() {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1385244461/issues/source?bid=2979575&fileBranchId=2980149#l21)